src/cork.rs

```diff
@@ -67,6 +59,7 @@ impl proto::contract_call_service_server::ContractCallService for CorkHandler {
             }
         };
         debug!("cork: {:?}", encoded_call);
+        info!("hex cork: {:?}", hex::encode(&encoded_call));
```

Comment:
I think this is leftover cruft we wouldn't want at the info level. Presumably this would replace the above debug message anyway?